sdk/keyvault/azure-keyvault-certificates/tests/test_examples_certificates.py

```diff
@@ -129,8 +129,21 @@ def test_example_certificate_list_operations(self, vault_client, **kwargs):
             san_dns_names=["sdk.azure-int.net"],
         )
 
+        polling_interval = 0 if self.is_playback() else 5
+
+        pollers = []
+
         for i in range(4):
-            certificate_client.begin_create_certificate(name="certificate{}".format(i), policy=cert_policy).wait()
+            pollers.append(
```

Comment:
FYI, totally optional, you can roll these `append` loops into list comprehensions, e.g. `pollers = [certificate_client.begin_create_certificate(...) for i in range(4)]`

Reply:
thanks, switched!